posts/Rust-1.85.1.md

```diff
@@ -0,0 +1,36 @@
++++
+layout = "post"
+date = 2025-03-18
+title = "Announcing Rust 1.85.1"
+author = "The Rust Release Team"
+release = true
++++
+
+The Rust team has published a new point release of Rust, 1.85.1. Rust is a
+programming language that is empowering everyone to build reliable and
+efficient software.
+
+If you have a previous version of Rust installed via rustup, getting Rust
+1.85.1 is as easy as:
+
+```
+rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][rustup] from the
+appropriate page on our website.
+
+[rustup]: https://www.rust-lang.org/install.html
+
+## What's in 1.85.1
+
+- [Fix the doctest-merging feature of the 2024 Edition.](https://github.com/rust-lang/rust/pull/137899/)
```

Comment:
At the moment, these bullets are the same as the relnotes, but we should probably say more about this one at least. @rust-lang/rustdoc would any of you like to write something here? It could perhaps be adapted from one of this issue statements about this, but targeting a more general audience.

Reply:
I just added more here.